style.css

```diff
@@ -367,3 +367,152 @@ h4 {
 
   color: #fff;
 }
+
+.form {
+  display: flex;
+}
+
+.form img {
+  position: absolute;
+  width: 74px;
+  height: 74.02px;
+  left: 0;
+  top: 4732px;
+}
+
+.form h1 {
+  position: absolute;
+  width: 249px;
+  height: 50px;
+  left: 16px;
+  top: 4770px;
+  font-family: "Playfair Display", "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+
+  /* identical to box height, or 50px */
+  color: #1a2236;
+}
+
+.form p {
+  position: absolute;
+  width: 288px;
+  height: 63px;
+  left: 16px;
+  top: 4868px;
+  font-family: Poppins, "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.form .form-start .name {
+  /* Rectangle 31 */
+
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 52px;
+  left: 16px;
+  top: 4960px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+}
+
+.form .form-start .email {
+  /* Rectangle 31 */
+
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 52px;
+  left: 16px;
+  top: 5020px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+}
+
+.form .form-start .message {
+  box-sizing: border-box;
+  position: absolute;
+  width: 330px;
+  height: 112px;
+  left: 16px;
+  top: 5082px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+}
```

Comment:
Nice job adding the contact form to your project 😄! Please consider the following observations to improve your design 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/207972773-ba4186b6-007a-4e35-8c32-e9493c0d12ed.png)  | ![image](https://user-images.githubusercontent.com/42799579/207972751-f2f292fe-6ab0-4d1f-be88-d713a29fe567.png)  |

- [ ] Make sure to add the corresponding padding to your inputs. Kindly notice that in the original design the texts have more distance between the input borders 😉.

- [ ] Please notice that in the original design there are labels with 'Your name' and 'Your email'. Make sure to add them to your project as well 😄.

- [ ] Please take into account that in your design the message field is centered.

- [ ] Kindly consider applying `border: 0` to your button 😉.

By updating this, your project will look closer to the original design! Remember: this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.